sapi/fpm/fpm/fpm_php.h

```diff
@@ -3,6 +3,7 @@
 #ifndef FPM_PHP_H
 #define FPM_PHP_H 1
 
+#include <stdbool.h>
```

Comment:
why do you need this? Isn't it already included in php.h ...?

Reply:
Should be yes, just because I prefer to be explicit but can remove it.

Reply:
yes please, remove it.

Reply:
it doesn't make sense to be done only for a single change - it would need to be applied everywhere otherwise it's just inconsistent.